lib/Cro/WebApp/Form.pm6

```diff
@@ -498,9 +498,11 @@ role Cro::WebApp::Form {
                 # Has to become:
                 #  YYYY-MM-DDTHH:MM:SS
                 my $ts = .Str;
-                my $cutoff = $ts.rindex(".") // $ts.rindex("-") // $ts.rindex("+");
-                my $i  = $ts.chars - $cutoff;
-                $ts .= substr(0, * - $i);
+                unless $ts.ends-with('Z') {
+                  my $cutoff = $ts.rindex(".") // $ts.rindex("-") // $ts.rindex("+");
+                  my $i  = $ts.chars - $cutoff;
+                  $ts .= substr(0, * - $i);
+                }
```

Comment:
Can you make the indentation consistent with the rest of the file? Thanks!

Reply:
Odd, it's differently wrong now (only the bit inside of the `unless` needed the fix). Feel free to take care of it, or I'll take care of it and manually merge later.

Reply:
I thought that was all I had changed. This is what I get for not properly configuring Atom. Wait one.

Reply:
Changes committed.